flink-python/pyflink/gen_protos.py

```diff
@@ -120,10 +120,17 @@ def _install_grpcio_tools_and_generate_proto_files():
     logging.warning('Installing grpcio-tools into %s', install_path)
     try:
         start = time.time()
-        subprocess.check_call(
-            [sys.executable, '-m', 'pip', 'install',
-             '--prefix', install_path, '--build', build_path,
-             '--upgrade', GRPC_TOOLS, "-I"])
+        pip_version = pkg_resources.get_distribution("pip").version
+        if pip_version >= '8.0.0':
```

Comment:
We can't do string compare here, how about use version.parse() to compare:
```
        from packaging import version
        if version.parse(pip_version) >= version.parse('8.0.0'):
```

Reply:
What about using `from pkg_resources import parse_version`? as we have already used module "pkg_resources"?

Reply:
+1. let's do it with `parse_version`.

Reply:
Done